apps/angular/src/app/app.component.html

```diff
@@ -1 +1,4 @@
-<router-outlet></router-outlet>
+<div class="container">
+  <camp-header class="header"></camp-header>
+  <anime-table class="table"></anime-table>
```

Comment:
We can use self closing tags in such cases
```suggestion
  <anime-table class="table" />
```